lib/ids.js

```diff
@@ -1,7 +1,10 @@
 const _ = require('underscore'),
       logger = require('./logger');
 
-const DEFAULT_ID_LENGTH = 5,
+const ID_LENGTH_DOC_ID = 'shortcode-id-length',
```

Comment:
If sentinel is already using a meta doc in other parts of the code we might also considering using it, or at least a similar naming schema so it's easy to find them all.  Then we cat fetch them all  with `{startkey: 'meta-sentinel-', endkey: 'meta-sentinel-\uffff'}` etc.